ncm-authconfig/pom.xml

```diff
@@ -1,37 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-
   <modelVersion>4.0.0</modelVersion>
-
   <groupId>org.quattor.cfg.module</groupId>
   <artifactId>authconfig</artifactId>
-
   <packaging>pom</packaging>
   <version>17.3.1-SNAPSHOT</version>
-  <name>authconfig NCM component</name>
-
-
+  <name>ncm-authconfig: authconfig NCM componen</name>
```

Comment:
Any objection to just making these the module name?
i.e. `<name>ncm-authconfig</name>`